src/luks2_notes.txt

```diff
@@ -0,0 +1,71 @@
+--pbkdf <PBKDF spec>
+   Set Password-Based Key Derivation Function (PBKDF) algorithm for LUKS keyslot.  The PBKDF
+   can be: pbkdf2 (for PBKDF2 according to RFC2898), argon2i for Argon2i or argon2id for Ar-
+   gon2id (see https://www.cryptolux.org/index.php/Argon2 for more info).
+
+   For  LUKS1, only PBKDF2 is accepted (no need to use this option).  The default PBKDF2 for
```

Comment:
Ok, I'm a bit lost here. Let's see your answers in previous comments I have left.

Reply:
Oh, this file was not intended to be uploaded!